prototypes/basic/app/views/review.html

```diff
@@ -4,21 +4,19 @@
 
 {% block pageTitle %} – {{ serviceName }} – GOV.UK Prototype Kit {% endblock %}
 {% block beforeContent %}
-    {{ govukBackLink({ text: "Back", href: "javascript:window.history.back()" }) }}
+{{ govukBackLink({ text: "Back", href: "javascript:window.history.back()" }) }}
 {% endblock %}
 
 {% block content %}
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-l">Review your data</h1>
 
-        {{ importerTableView(data) }}
-
```

Comment:
Why?? I just have a blank screen on that page now.

Reply:
Because we've changed how we provide data to that function now, and so we'll need to wrap it in another macro (which will fetch the mapped data) so that the user doesn't have to call functions themselves.  And given we discussed removing that preview from that page and adding warnings and showing the source data, it seemed like it would be wasted effort to fix that right now

Reply:
OK – can you take the page out of the `prototypes/basic` for now then, and just go straight to the upload complete page? The issue for me is that if I give a demo it just looks a bit off.

Reply:
Removed, the mapping page goes to success now, will re-instate in line with what we discussed around showing warnings.